website/content/docs/concepts/filtering/audit.mdx

```diff
@@ -1,165 +1,160 @@
+filtered out and dropped from the audit record. Enabling an audit device with
+the `fallback` parameter ensures that Vault continues to adhere to the default
+[security model](/vault/docs/internals/security) which mandates that all
+requests and responses must be successfully logged before clients receive secret
+material.
+
+<Warning title="Use a fallback to avoid missing audit entries">
+
+  For Vault installations that use filtered audit devices **exclusively**,
+  configuring a fallback audit device is the only way to guarantee a comparable
+  security standard as Vault installations that only use standard, non-filtered
+  audit devices.
+
+</Warning>
+
+### Fallback telemetry metrics
+
+When the fallback device successfully writes an audit entry to the audit log,
+Vault emits a
+[fallback 'success' metric](/vault/docs/internals/telemetry/metrics/audit#vault-audit-fallback-success).
+
+If you enable filtering **without** a fallback device, Vault emits a
+[fallback 'miss' metric](/vault/docs/internals/telemetry/metrics/audit#vault-audit-fallback-miss)
+metric anytime an audit entry would have been written to the fallback so you can
+track how many auditable events you have lost.
+
+## Audit device limitations
+
+1. You cannot add filtering to an existing audit device.
+1. You can configure filtering when enabling one of the following supported audit device types:
+    - [file](/vault/docs/audit/file)
+    - [socket](/vault/docs/audit/socket)
+    - [syslog](/vault/docs/audit/syslog)
+1. You can only designate one auditing fallback device.
+
+## Filtering and test messages
+
+By default, Vault sends a test message to the audit device when you enable it.
+Depending on how you configure your filters, the default test message may fail
+the predicate expression and not write to the new device.
+
+You can determine whether the test message should appear in the sink for the 
+newly enabled audit device based on the following property
+table, which are common to all default test messages.
+
+Property      | Value
+------------- | ----------------
+`mount_point` | empty
+`mount_type`  | empty
+`namespace`   | empty
+`operation`   | `update`
+`path`        | `sys/audit/test`
+
+## `filter` properties for audit devices
+
+Filters can only reference the following properties of an audit entry:
+
+Property      | Example                             | Description
+------------- | ----------------------------------- | --------------------------
+`mount_point` | `mount_point == auth/oidc`          | Log all entries for the `auth/oidc` mount point
+`mount_type`  | `mount_type == \"kv-v2\"`           | Log all entries from `kv-v2` plugins
+`namespace`   | `namespace != admin/`               | Log all entries **not** in the admin namespace
+`operation`   | `operation == read`                 | Log all read operations
+`path`        | `path == auth/approle/login`        | Log all activity against the AppRole login path
```

Comment:
We should encourage users to escape quote the values (I think you've mentioned this in a comment already to make life easier when folks copy/paste).

```suggestion
`mount_point` | `mount_point == \"auth/oidc\"`          | Log all entries for the `auth/oidc` mount point
`mount_type`  | `mount_type == \"kv-v2\"`           | Log all entries from `kv-v2` plugins
`namespace`   | `namespace != \"admin/\"`               | Log all entries **not** in the admin namespace
`operation`   | `operation == \"read\"`                 | Log all read operations
`path`        | `path == \"auth/approle/login\"`        | Log all activity against the AppRole login path
```